### BIG differences, this is diff of wrong files.
### CLUS "Type Specifier VALUES"
### and CLHS topic selection http://clhs.lisp.se/Body/a_values.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:values.txt	2017-04-27 21:39:30.000000000 +0100
+++ saved-pages-clus/cl:types:values.txt	2017-04-27 21:39:30.000000000 +0100
@@ -1,4 +1,32 @@
-VALUES
-Please select which reference to VALUES you intended
+Type Specifier VALUES

-AccessorType Specifier
\ No newline at end of file
+Compound Type Specifier Kind
+Specializing.
+Compound Type Specifier Syntax
+values value-typespec
+value-typespec = typespec* [&optional typespec*] [&rest typespec] ['&allow-other-keys]
+</blockquote>
+====Compound Type Specifier Arguments====
+typespec - a type specifier.
+====Compound Type Specifier Description====
+This type specifier can be used only as the value-type in a function type specifier or a the special form. It is used to specify individual types when multiple values are involved. The &optional and &rest markers can appear in the value-type list; they indicate the parameter list of a function that, when given to multiple-value-call along with the values, would correctly receive those values.
+The symbol * may not be among the value-types.
+The symbol values is not valid as a type specifier; and, specifically, it is not an abbreviation for (values)''.
+\
\ No newline at end of file
